### ISSUE TITLE
grafana links missing

### ISSUE THREAD

**Issue by jmazzitelli** [kiali/kiali#4226]
There are no Grafana links in the UI that I can find - other than in the about box (which takes me to a valid URL to Grafana). There are no messages in the Message Center indicating a problem.

Here's the About box:

![image](https://user-images.githubusercontent.com/2029470/127225830-bbc8a01e-128b-40b0-a737-f9ae62a5f157.png)

Here's where I think the Grafana links should be - shouldn't there be links at the top somewhere?:

![image](https://user-images.githubusercontent.com/2029470/127225247-a06740c8-ff62-4f20-9170-d34f1e4f349b.png)

Only thing odd in the logs is this:
```
2021-07-27T20:36:47Z INF Cluster ID couldn't be resolved. Most likely, no Cluster ID is set in the service mesh control plane configuration.
```

**Comment by prachiyadav** [comment]
Test case >> SWSQE-89

**Comment by leandroberetta** [comment]
Fixed by #4236.
